### PR TITLE
Transient ReferenceSummary on ApprovalRequest

### PR DESCRIPTION
**Pull request by tharmaman**
This property is solely used on the client, therefore just needs to use a tableCellFormatter to provide the preview of the viewReference.

### CODE REVIEW

File: src/foam/nanos/approval/ApprovalRequest.js [hunk 1]
```diff
@@ -138,6 +138,25 @@
   ],
 
   properties: [
+    {
+      class: 'String',
+      name: 'referenceSummary',
+      section: 'requestDetails',
```
Comment (nanoscott) [inline src/foam/nanos/approval/ApprovalRequest.js]: can you update the section here to be 'approvalRequestInformation' and set the order to be 21 and the gridColumns to be 6

File: src/foam/nanos/approval/ApprovalRequest.js [hunk 2]
```diff
@@ -138,6 +138,25 @@
...
+      transient: true,
+      tableWidth: 200,
+      tableCellFormatter: function(_,obj) {
```
Comment (nanoscott) [inline src/foam/nanos/approval/ApprovalRequest.js:133]: the detailed view also has to be set, not only the table cell

**Review by nanoscott — CHANGES_REQUESTED** [review]

**Review by nanoscott — APPROVED** [review]

File: src/foam/nanos/approval/ApprovalRequest.js (hunk 2 above)
Comment (nanoscott) [reply src/foam/nanos/approval/ApprovalRequest.js:133]: we can do this in a followup

File: src/foam/nanos/approval/ApprovalRequest.js (hunk 1 above)
Comment (tharmaman) [reply src/foam/nanos/approval/ApprovalRequest.js]: Done.

File: src/foam/nanos/approval/ApprovalRequest.js (hunk 2 above)
Comment (tharmaman) [reply src/foam/nanos/approval/ApprovalRequest.js:133]: Done.

**Pull request merged**